pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go

```diff
@@ -291,6 +311,11 @@ func (pl *nonCSILimits) filterVolumes(volumes []v1.Volume, namespace string, fil
 				continue
 			}
 
+			// ... the PVC for an ephemeral volume must be owned by the pod.
+			if ephemeral && !metav1.IsControlledBy(pvc, pod) {
```

Comment:
same as above, this could be moved to the switch case

Reply:
Same here - it needs `pvc`  :grin: